### DISCUSSION TITLE
Sum of Bills/Subscriptions - Layout v2

### DISCUSSION THREAD

**Discussion by JanFeld** [firefly-iii/firefly-iii#5128] [Firefly III]
Hi,

maybe i'm blind but i can't find the sum of bills per month for each group and the sum of all in layout v2.
Is this feature implemented yet?

Cheers,
Jan

**Comment by JC5** [comment]
Probably not yet 😅

**Reply by JanFeld** [reply]
At least i'm not blind! 

Actually that's something i really appreciate in the overview of bills.

By the way, i've noticed the bill-overview shows a date two months ago (july) in the _Payment Info_ field.
I'm not sure if this is related to the date-issue mentioned in another thread/issue.
